README.md

```diff
@@ -45,18 +43,59 @@ Currently, only `linux/amd64` with `AVX` is supported.
   - `--n_out_of_memory_merkle_layers`
   - `--table_prover_n_tasks_per_segment`
 
+### Prove bootloader
+
+- Generate a proof for the bootloader Cairo program
+- `stone-cli prove --cairo_program <program-path>`
+- Additional args:
+  - `--program_input`
+  - `--program_input_file`
+  - `--layout`
+  - `--prover_config_file`
+  - `--parameter_file`
+
 ### Verify
 
 - Verify a proof generated by the prover
 - `stone-cli verify --proof <proof-path>`
+- Additional args:
+  - `--annotation_file`
+  - `--extra_output_file`
 
 ### Serialize Proof
 
 - Serialize a proof to a file
 - `stone-cli serialize-proof --proof <proof-path> --network <network> --output <output-path>`
+- Additional args:
+  - `--annotation_file`
+  - `--extra_output_file`
 
 Using `--network starknet` serializes the Cairo proof into a format that can be verified on the Cairo verifier deployed on Starknet. Please refer to the [integrity documentation](https://github.com/HerodotusDev/integrity) for more information on how to use the calldata to send a transaction to Starknet.
 
+Using `--network ethereum` serializes the Cairo proof into a format that can be verified on the Cairo verifier deployed on Ethereum. Please refer to the [the next section](#how-to-create-proofs-and-verify-them-on-ethereum) for more information on how to create proofs that can be verified on Ethereum.
+
+### How to create proofs and verify them on Ethereum
+
+Currently there is a Cairo verifier deployed on Ethereum, which are mainly used to verify SHARP proofs created by L2 Starknet nodes. The Cairo verifier checks the validity of a Cairo program named `bootloader`, which recursively verifies multiple Cairo programs or a Cairo PIEs (Position Independent Executable) and allows a single proof to prove executions of multiple Cairo programs. Once we create a bootloader proof, we need to serialize it to a format that works for the Cairo verifier on Ethereum.
```

Comment:
"recursively verifies". 

My understanding is that that the bootloader just hashes the program and jumps to it?